python/servo/platform/linux.py

```diff
@@ -7,11 +7,12 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+import distro
 import os
 import subprocess
 from typing import Optional, Tuple
 
-import distro
+from ..build_target import BuildTarget
```

Comment:
Would it make sense to move `BuildTarget` into `platform` to avoid having this module depend on things from the parent?

Reply:
Done in [6245f0b](https://github.com/servo/servo/pull/33114/commits/6245f0b30541dcff311902e33cfc44b8518139eb).